src/index.html

```diff
@@ -11,7 +11,182 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form"
+    >
+      <fieldset class="form__fieldset">
+        <legend class="form__fieldset-legend">Personal information</legend>
+        <label class="form__lable">
+          Surname:
+          <input
+            name="surname"
+            type="text"
+            id="surname"
+            class="form__input"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__lable">
+          Name:
+          <input
+            name="name"
+            type="text"
+            id="name"
+            class="form__input"
+            maxlength="30"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form__lable">
```

Comment:
add an empty line between _multiline_ siblings
```suggestion
        </label>
        
        <label class="form__lable">
```